doc/bridged_USDC_standard.md

```diff
@@ -16,22 +16,46 @@ This document provides a high-level overview of the process. Note that this also
    - Circle and the third-party team will jointly coordinate to burn the supply of native USDC locked in the bridge contract on the origin chain and upgrade the bridged USDC token contract on the destination chain to native USDC.
 4. The native USDC token contract seamlessly retains the existing supply, holders, and app integrations of the original bridged USDC token contract.
 
-### Bridge Contracts
+## Bridge Contracts
 
-The third-party team’s bridge contracts must be upgradeable to add required, specific functionality at a later point in time to support the upgrade steps above, once Circle and the third-party team jointly decide to perform the upgrade.
+The third-party team’s bridge contracts play an integral role and require small additions to support the upgrade process. 
```

Comment:
Q: are we intentionally removing source bridge upgradeability as a requirement?
If not, I feel like this is still an important detail to call out. We should considering specifying the defined interface for `burnAllLockedUSDC` as well.

Reply:
The requirement is specified below on `burn`, but my understanding is that this doc intends to clarify things before they start deploying, and we recommend them to add the burn functionality after we mutually agree. So they aren't missing out this particular function name particularly. The following two lines hinted some on this regard.